.github/workflows/test_api_variant_on_custom_api_project.yml

```diff
@@ -0,0 +1,49 @@
+name: "Test API variant on custom API project"
+
+on: push
+
+env:
+  OTP_VERSION: 23.1.1
+  ELIXIR_VERSION: 1.11.0
+  PHOENIX_VERSION: 1.5.5
+
+jobs:   
+  unit_test:
+    runs-on: ubuntu-latest
+    
+    services:
+      db:
+        image: postgres:12.3
+        ports: ['5432:5432']
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      - uses: actions/setup-elixir@v1
+        with:
+          otp-version: ${{ env.OTP_VERSION }}
+          elixir-version: ${{ env.ELIXIR_VERSION }}
+          
+      - name: Cache Elixir build
+        uses: actions/cache@v2
+        with:
+          path: |
+            _build
+            deps
+          key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-mix-
+              
+      - uses: nimblehq/elixir-templates@bug/fix-github-action
```

Comment:
The composite action looks great but then what is your plan to have the branch `@bug/fix-github-action` changed? 😆

Reply:
I tried to use the current branch with [${GITHUB_REF#refs/heads/}](https://stackoverflow.com/questions/58033366/how-to-get-current-branch-within-github-actions) but that doesn't work as the `users` doesn't support it.

So my plan will open another PR to change this to `develop` once this one is merged.